sap/adt/aunit.py

```diff
@@ -297,6 +303,14 @@ def is_warning(self):
 
         return self.severity == AlertSeverity.TOLERABLE
 
+    @property
+    def is_skipp(self):
```

Comment:
Why `is_skipp` instead of `is_skipped`?

Reply:
Ah, I know. The class is Alert, the method says if it is an alert saying something was skipped. But the alert itsefl is not skipped. Using the name "is_skipped" would look like that the alert was skipped. Does that make sense?

Reply:
It's minor thing, I just saw `is_skipp`, so i tought it should be `is_skipped` or `is_skip`.

Reply:
:D - uh, oh, eh - I didn't notice the typo ..